### PR TITLE
Nuke the syndicate base

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I'm not sure how often this will ever happen but it will be fun if it does.

:cl: Kor
rscadd: The game will now recognize a successful detonation of the nuclear bomb in the syndicate base.
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
:3c

**Comment by Ultimate-Chimera** [comment]
wat

**Comment by Jalleo** [comment]
Wasnt there a old bug report about it breaking the server?

**Comment by KorPhaeron** [comment]
That got fixed by tkdrg or incoming, one of the two

**Comment by ExcessiveUseOfCobblestone** [comment]
does this not encourage people to """""find""""" the syndicate ship and try to counter raid it for OMEGA-REDTEXT

**Comment by KorPhaeron** [comment]
probably

**Comment by Durkel** [comment]
pls don't encourage the meta shitters who would go look for the ship.

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -195,7 +195,12 @@
 	//derp //Used for tracking if the syndies actually haul the nuke to the station	//no
 	//herp //Used for tracking if the syndies got the shuttle off of the z-level	//NO, DON'T FUCKING NAME VARS LIKE THIS
 
-	if      (!disk_rescued &&  station_was_nuked && !syndies_didnt_escape)
+	if(nuke_off_station == 3)
+		feedback_set_details("round_end_result","loss - syndicated nuked - disk secured")
```
Comment (GunHog) [inline code/game/gamemodes/nuclear/nuclear.dm]: syndicate*

**Comment by InsaneHyena** [comment]
Do normies even have access to syndicate navigation computer?

**Comment by KorPhaeron** [comment]
@InsaneHyena 

No, but every once in a while an op fucks up spectacularly and you take his ID card

**Comment by PKPenguin321** [comment]
I think I've actually done this before so :+1:

**Comment by kilkun** [comment]
whats to stop the nuke ops from collaborating with the station to blow up the syndicate fob and secede from the syndicate?

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 2]
```diff
@@ -195,7 +195,12 @@
 	//derp //Used for tracking if the syndies actually haul the nuke to the station	//no
 	//herp //Used for tracking if the syndies got the shuttle off of the z-level	//NO, DON'T FUCKING NAME VARS LIKE THIS
 
-	if      (!disk_rescued &&  station_was_nuked && !syndies_didnt_escape)
+	if(nuke_off_station == 3)
```
Comment (AnturK) [inline code/game/gamemodes/nuclear/nuclear.dm]: Please turn these into defines while you're here.

**Review by AnturK — CHANGES_REQUESTED** [review]

**Comment by KorPhaeron** [comment]
>conflicts

antur betrayed me

**Pull request closed without merging**